packages/xo-server/config.toml

```diff
@@ -59,6 +59,8 @@ listingDebounce = '1 min'
 # It should be short to avoid blocking the display of the available backups.
 vmBackupSizeTimeout = '2 seconds'
 
+poolMetadataTimeout = '2 seconds'
```

Comment:
It really enough?

It looks really short, what does it need to be so short?

Reply:
I've done some tests  with 1 second and they pass i can increase it to `10 seconds`.

Reply:
It's not because you did a small test with 1s that it will pass with 2s, it might depends of the size of the pool, the load on the pool master or even the load on XO…

A timeout is a compromise between the time a function can take and the time we are willing to wait for it to finish.

Here, you have done a test and it seems to run in a second, and there is no special need to fail fast, so it would make sense to use a timeout around 10 mins.